packages/client/src/pages/PortalSettings/categories/integration/ThirdPartyServicesSettings/sub-components/consumerModalDialog.js

```diff
@@ -237,6 +240,7 @@ class ConsumerModalDialog extends React.Component {
           <Button
             primary
             size="normal"
+            id="enable"
```

Comment:
Too simple id value (potential duplicates)

Reply:
Fix to "enable-button"